extension/tailwind.config.js

```diff
@@ -0,0 +1,85 @@
+module.exports = {
```

Comment:
Added only required config for current scope. Will add more preset styles based on the Quill Design system as required.